cronjob_update_build_run.sh

```diff
@@ -0,0 +1,37 @@
+repo="https://github.com/FelixStegmeier/wicked_migration_server.git"
+server_dir="./wicked_migration_server"
+migration_dir="$server_dir/wicked_migration_server"
+
+git_has_changes() {
+	main=$(git rev-parse main)
+	origin_main=$(git rev-parse origin/main)
```

Comment:
This will call git in the current working directory, which isn't actually the git directory.